jobs/pacman-rule-engine-2.0/src/main/java/com/tmobile/pacman/executor/PolicyExecutor.java

```diff
@@ -318,12 +318,9 @@ private void run(Map<String, String> policyParam, String executionId)
             logger.debug("total evaluations received back from policy runner:" + evaluations.size());
         } catch (Exception e) {
             String msg = "error occurred while executing";
-            logger.error(msg, e);
             policyEngineStats.put(msg, Strings.isNullOrEmpty(e.getMessage()) ? "" : e.getMessage());
-            logger.error(DATA_ALERT_ERROR_STRING + JOB_NAME + " with job id "
-                    + policyParam.get(PacmanSdkConstants.POLICY_UUID_KEY)
-                    + "Exception occurred for policy with policyId:" + policyParam.get(PacmanSdkConstants.POLICY_ID));
-            logger.error("exiting now.." + e);
+            logger.error(msg, e);
+            logger.info("exiting now..");

```

Comment:
not needed, meaningless log since you are already logging an error and returning

Reply:
got it, removed